src/Analyzers/Analyzers.csproj

```diff
@@ -9,7 +9,7 @@
     <IncludeBuildOutput>false</IncludeBuildOutput>
     <SuppressDependenciesWhenPacking>true</SuppressDependenciesWhenPacking>
     <IncludeShared>false</IncludeShared>
-    <GenerateDocumentationFile>false</GenerateDocumentationFile>
+    <GenerateDocumentationFile>true</GenerateDocumentationFile>
```

Comment:
Why do we want to generate the documentation file for the analyzer? I wouldn't expect that we'd want to since we wouldn't normally expect developers to program against the analyzer's APIs, would we?

Reply:
StyleCop was producing a warning without it. Figured it was easier to just generate it then dig into that warning.